test/integration/analysis-controller-existing-status/install.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: AnalysisValueTemplate
+metadata:
+  name: ready
+  namespace: $NAMESPACE
+spec:
+  provider:
+    name: my-mocked-provider
+    namespace: $NAMESPACE
+  query: 'sum(kube_pod_container_status_ready{namespace="{{.ns}}"})'
+---
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: AnalysisDefinition
+metadata:
+  name: ed-my-proj-dev-svc1
+  namespace: $NAMESPACE
+spec:
+  objectives:
+    - analysisValueTemplateRef:
+        name: ready
+        namespace: $NAMESPACE
+      target:
+        failure:
+          lessThan:
+            fixedValue: 2
+        warning:
+          lessThan:
+            fixedValue: 3
+      weight: 1
+      keyObjective: false
+  totalScore:
+    passPercentage: 90
+    warningPercentage: 75
+---
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: Analysis
+metadata:
+  name: analysis-sample
+  namespace: $NAMESPACE
+spec:
+  timeframe:
+    from: 2023-09-14T07:33:19Z
+    to: 2023-09-14T07:33:19Z
+  args:
+    "ns": "keptn-lifecycle-toolkit-system"
+  analysisDefinition:
+    name: ed-my-proj-dev-svc1
+    namespace: $NAMESPACE
+status:
```

Comment:
i think applying this will not actually set the status, as `kubectl apply` omits the status if i recall correctly - but we can keep this test for now and I can try to extend this as I'm currently working on adding further integration tests

Reply:
I did expect kubebuilder to fail... we can try to have mock server installed after the apply to have a fail and later have it removed.. I will play with it